sal_interfaces/Scheduler/Scheduler_Commands.xml

```diff
@@ -36,4 +36,72 @@
     <EFDB_Topic>Scheduler_command_computePredictedSchedule</EFDB_Topic>
     <Description>This command causes the scheduler to compute and publish the predicted schedule. It can only run if the scheduler is idle (resume command not sent). This runs automatically as part of the advance operation mode so this command is only useful when users want to estimate what is ahead when the scheduler is not running.</Description>
   </SALCommand>
+  <SALCommand>
+    <Subsystem>Scheduler</Subsystem>
+    <EFDB_Topic>Scheduler_command_addBlock</EFDB_Topic>
+    <Description>Add new block to the scheduling queue.</Description>
+    <item>
+      <EFDB_Name>id</EFDB_Name>
+      <Description>The name of the block (e.g., OBS-123).</Description>
+      <IDL_Type>string</IDL_Type>
+      <IDL_Size>1</IDL_Size>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>override</EFDB_Name>
+      <Description>Override information in the block with those provided here.</Description>
```

Comment:
Issue here is that it doesn't say what the input is... which is the JSON string..
Maybe something like:
"Override block with a new JSON string provided here"  ?
I don't like that much either honestly....

Reply:
I would rather not tie in with a particular solution (e.g. JSON) here.. Just in case we want to change to some other format later. In the CSC codebase I will document what should be the format.